src/openassetio-python/package/openassetio/pluginSystem/PythonPluginSystemManagerImplementationFactory.py

```diff
@@ -37,48 +37,97 @@ class PythonPluginSystemManagerImplementationFactory(ManagerImplementationFactor
     derived plugins. Not usually used directly by a @ref host, which
     instead uses the @fqref{hostApi.ManagerFactory} "ManagerFactory".
 
+    The factory loads plugins from exposed `openassetio.manager_plugin`
+    entry points, as well as additional plugins found under paths
```

Comment:
What do you think of:
```suggestion
    The factory loads plugins from entry points registered within the `openassetio.manager_plugin`
    entry point group, as well as additional plugins found under paths
```
?